### ISSUE TITLE
RyuJIT x86: NYI: "Binary operator with GTF_REVERSE_OPS"

### ISSUE THREAD

**Issue by adiaaida** [dotnet/coreclr#4593]
This NYI:

NYI: "Binary operator with GTF_REVERSE_OPS"
File: coreclr\src\jit\lower.cpp Line: 413

Is hit by these tests:

JIT\Methodical\int64\signed\_speed_rels_addsub\_speed_rel
JIT\Regression\CLR-x86-JIT\V1-M11-Beta1\b36274\b36274\b36
JIT\Methodical\int64\signed\_speed_dbgs_addsub\_speed_dbg
JIT\Methodical\int64\signed\_dbgs_addsub\_dbgs_addsub.cmd
JIT\Methodical\int64\signed\_il_rels_addsub\_il_rels_adds
JIT\Methodical\int64\signed\_rels_addsub\_rels_addsub.cmd

**Comment by mikedn** [comment]
Corrected test paths:
```
JIT\Methodical\int64\signed\_speed_rels_addsub\_speed_rels_addsub.cmd
JIT\Regression\CLR-x86-JIT\V1-M11-Beta1\b36274\b36274\b36274.cmd
JIT\Methodical\int64\signed\_speed_dbgs_addsub\_speed_dbgs_addsub.cmd
JIT\Methodical\int64\signed\_dbgs_addsub\_dbgs_addsub.cmd
JIT\Methodical\int64\signed\_il_rels_addsub\_il_rels_addsub.cmd
JIT\Methodical\int64\signed\_rels_addsub\_rels_addsub.cmd
```

**Comment by mikedn** [comment]
It looks like there's nothing to do about `GTF_REVERSE_OPS` if temporaries are introduced like in #5884. Temporaries can be loaded in any order and the `GTF_REVERSE_OPS` does not need to be propagated to the new lo/hi nodes.

**Comment by mikedn** [comment]
@RussKeldorph You can assign this to me as #5884 will fix this as well.

**Comment by RussKeldorph** [comment]
@mikedn Thanks!

**Comment by BruceForstall** [comment]
@mikedn This was fixed with the LIR change #6689. Can I (or you) close the issue?

**Comment by mikedn** [comment]
Looks like you need to close it, I can only close issues that I created.
